### ISSUE TITLE
Add new menu button for Frigate exports

### ISSUE THREAD

**Issue by kirsch33** [dermotduffy/frigate-hass-card#1277]
It would be really cool if frigate-hass-card could have a new menu button that is linked to the Frigate exports folder. This would function just like the snapshot/clips viewer in my mind, and would allow viewing of timelapses and other exports right in the HA app. 

This would be made better with the ability to navigate directly to a specific file via query string card action as well. 

The idea behind this is using the new Frigate Timelapse API to automate a timeline export on an interval and then send a mobile notification when the export is done. Clicking on the notification would then launch directly into the card to the export in the media viewer.

**Comment by dermotduffy** [comment]
I haven't played (at all) with the export functionality. Is the media you export already available in HA? If yes, I think this is probably possible for the card to fetch via the HA media browser API, but as the events/recordings are natively fetched from the Frigate backend (rather than relying on the HA media browser) I'd want to see if there was a similar API call to fetch exports. I could imagine this implemented as an entirely new kind of media (e.g. we have `clips`, `snapshots` and `recordings` today -- maybe we need something new). A quick look at the backend code and I don't see an API call like this available right now.

Bottom line: I think this is a valid feature request, but probably not easy to implement and possibly requiring changes beyond the card.

More thumbs up => more chance I'll work on it!

**Comment by kirsch33** [comment]
i may be misunderstanding but I think they can be fetched just like events/recordings, see this comment from @NickM-27 

https://github.com/blakeblackshear/frigate/discussions/7936#discussioncomment-7122408

**Comment by NickM-27** [comment]
https://github.com/blakeblackshear/frigate/blob/d11c1a20668bca48413fe37895d905b854bc675b/docker/main/rootfs/usr/local/nginx/conf/nginx.conf#L175-L194

**Comment by dermotduffy** [comment]
> autoindex_format json;

OK!

> i may be misunderstanding but I think they can be fetched just like events/recordings, see this comment from @NickM-27

It's not the same. For events and recordings, there's a dedicated API endpoint on the backend that returns event & recording metadata that the card calls. Here this looks like it will give a standard directory index dump, but conveniently in JSON. So it can be fetched, but it's in a different way. No big deal, just more code.
